language_server/src/lsp/lsp_server.h

```diff
@@ -41,6 +43,11 @@ class server final : public hlasm_plugin::language_server::server,
     // Parses LSP (JSON RPC) message and calls corresponding method.
     void message_received(const json& message) override;
 
+    // Inherited via json_sink
+    // Takes json with telemetry info and sends it through LSP
+    void write(const nlohmann::json&) override;
+    void write(nlohmann::json&&) override;
+
```

Comment:
wouldn't it be better to have a dedicated interface that takes a telemetry_info type?